### DISCUSSION TITLE
Some basic questions about installation and api

### DISCUSSION THREAD

**Discussion by stratboy** [spatie/laravel-medialibrary#3515] [Q&A]
Hi, I cannot find anywhere here or in docs, about the following:

- how exactly am I going to install the media library? What's the exact package?
- what exactly does the addMedia() method?
- and what exactly does the toMediaCollection() method?

In general, docs do not include api docs. It would be great at least for the most use methods. There are examples, but no doc explains what thos methods really do, from a technical point of view.

**Comment by freekmurze** [comment]
You'll find the docs here, including installation instructions: https://spatie.be/docs/laravel-medialibrary/v11/installation-setup

Please do not open duplicate discussion.

**Reply by stratboy** [reply]
Great, thank you
